pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -204,3 +204,24 @@ func isAntreaPolicyEgressTable(tableID uint8) bool {
 	}
 	return false
 }
+
+// getPacketInTableID returns the OVS table ID in which the packet is sent to antrea-agent. Since L2ForwardOutput is
+// the table where all ANP logging packets are sent to antrea-agent, "PacketInTableField" is used to store the real
+// table requiring "sendToController" action. This function firstly parses the direct table where the packet leaves
+// OVS pipeline, then checks whether "PacketInTableField" is set with a valid value or not. The value in the field
+// is returned if yes.
+func getPacketInTableID(pktIn *ofctrl.PacketIn) uint8 {
+	tableID := pktIn.TableId
+	matchers := pktIn.GetMatches()
+	if match := getMatchRegField(matchers, openflow.PacketInTableField); match != nil {
+		tableVal, err := getInfoInReg(match, openflow.PacketInTableField.GetRange().ToNXRange())
+		if err == nil {
+			if tableVal == 0 || tableVal == 255 {
```

Comment:
Which scenario could lead to this be true? Should we throw an error instead?

Reply:
In previous test, I saw it because an issue existing when unmarshaling PacketIn2 message. So I left this branch. Maybe I should remove it since the issue was fixed now.